### ISSUE TITLE
Quarto in VSCode does not render a document

### ISSUE THREAD

**Issue by Tsumiyama-Isao** [quarto-dev/quarto#222]
When trying to render a document by Quarto using VSCode, the following pop-up appears but no document is rendered.

![image](https://github.com/quarto-dev/quarto/assets/56852320/77d3c7a4-4ef3-4d13-a309-8a5e4fe5f8d7)

Sorry for a Japanese message. This pop-up tells me to select an application to open a text dump. The dump seems produced by redirect.

Thanks in advance

**Comment by cderv** [comment]
Sorry that you are encountering this. However, there is really too little information in your issue above to try understand what can go wrong. 

VSCODE Extension and the render button works fine - I can produce document with it and no other report of failure currently. This means this is something related to your specific usage and you need to share more. 

* What is the quarto version ? 
* What is the content of the document ? 
* What is your setup ? 
* Are you using a project ? A single document ? 
* Does it works outside of VSCODE ? 

Basically, any information that we would need in order to help you. Without that, we won't be able to do much.
